src/containers/gui.jsx

```diff
@@ -25,14 +25,22 @@ import {
 } from '../reducers/modals';
 
 import FontLoaderHOC from '../lib/font-loader-hoc.jsx';
+import LocalizationHOC from '../lib/localization-hoc.jsx';
 import ProjectFetcherHOC from '../lib/project-fetcher-hoc.jsx';
 import ProjectSaverHOC from '../lib/project-saver-hoc.jsx';
 import vmListenerHOC from '../lib/vm-listener-hoc.jsx';
 import vmManagerHOC from '../lib/vm-manager-hoc.jsx';
-import {defaultProjectTitleMessages} from '../reducers/project-title';
 
 import GUIComponent from '../components/gui/gui.jsx';
 
+const messages = defineMessages({
```

Comment:
Just a note, Chris and I agreed that when I/we refactor gui.jsx to extract projectId and projectTitle handling (see https://github.com/LLK/scratch-gui/issues/3430), this message should probably come with that logic